src/extension.ts

```diff
@@ -229,11 +229,8 @@ async function activate(context: vscode.ExtensionContext) {
     vscode.commands.registerCommand('calva.selectCurrentForm', select.selectCurrentForm)
   );
   context.subscriptions.push(
-    vscode.commands.registerCommand('calva.loadFile', async () => {
-      await eval.loadFile({}, config.getConfig().prettyPrintingOptions);
-      return new Promise((finished) => {
-        outputWindow.appendPrompt(finished);
-      });
```

Comment:
The prompt is already appended when the file is loaded, so there's no need to do it here, which actually causes it to be printed twice.